api/utils/helpers.py

```diff
@@ -1,9 +1,91 @@
-from flask import json, Response, request
+import json
+import os
+import logging
+from types import GeneratorType
+
 from sqlalchemy_utils import sort_query
 
 
+def responsify(body, code, headers=None):
+    default_headers = {"Access-Control-Allow-Origin": "*", "Access-Control-Allow-Credentials": True}
+    default_headers.update(headers) if headers else None
+    results_obj = {}
+    if isinstance(body, dict):
+        results_obj = body
+    elif isinstance(body, list) or isinstance(body, str):
+        results_obj = {"data": body}
+    elif isinstance(body, GeneratorType):
+        results_obj = {"data": [item for item in body]}  # Unpack generator
+    return {
+        "statusCode": code,
+        "isBase64Encoded": False,
+        "headers": default_headers,
+        "body": json.dumps(results_obj)
+    }
+
+
+class Abort(Exception):
+    def __init__(self, code, reason):
+        msg = f"ERROR:{code} MSG: {reason}"
+        super(Abort, self).__init__(msg)
+
+
+class Serverless:
+    logger = logging.getLogger()
+    logger.setLevel(logging.INFO)
+    args = {}
+
+    class Args(object):
+        def __init__(self, parameters={}):
+            self.params = parameters
+
+        def get(self, key, default, type=str):
+            try:
+                value = self.params.get(key)
+                if value:
+                    return type(value)
+                else:
+                    return default
+            except TypeError:
+                raise Abort(400, f"{type} is not a valid Type")
+            except ValueError:
+                return Abort(400, f"Invalid literal {key} for type {type}")
```

Comment:
Query params should be returned as the default value if you fail to convert using `type` instead of returning a 400 error